src/App.js

```diff
@@ -13,12 +14,16 @@ class App extends Component {
   state = {
     value: '',
   }
-  addTask = () => Boolean(this.state.value.trimRight()) && !this.props.tasks.includes(this.state.value) ? this.props.addTask(this.state.value) : null
+  addTask = () =>{
+    const { value } = this.state
+    return !isEmpty(value) && !this.props.tasks.includes(value) ? this.props.addTask(value) : null
```

Comment:
dont you think, that there should be some error messages showed to the user, if we are not allowing adding the same value? For empty string only, it would be enough... So - please add form validation and error showing to the user.